inventories/sip3-captain.yml

```diff
@@ -0,0 +1,34 @@
+all:
+  children:
+    captain:
+      hosts:
+        pbx1:
+          ansible_host: 23.08.20.15
+        pbx2:
+          ansible_host: 29.11.19.88
+          captain:
+            pcap:
+              dev: eth1
+  vars:
+    ansible_user: root
+    ansible_port: 22
+    ansible_become: true
+
+    platform: centos
+    edition: ce
+    version: 2020.1.1
+
+    features:
+      - rtcp
+
+    captain:
+      pcap:
+        dev: eth0
+        bpf_filter: udp
+      sender:
+        uri: udp://192.168.1.2:15060
+      management:
+        uri: udp://192.168.1.2:15090
```

Comment:
We need to fix usage of this property [here](https://github.com/sip3io/sip3-ansible/blob/nginx-and-2020.2.1/roles/captain/templates/application.yml.j2#L94).

Reply:
And also change default `pcap.bpf-filter` value [here](https://github.com/sip3io/sip3-ansible/blob/nginx-and-2020.2.1/roles/captain/templates/application.yml.j2#L21) to `udp or tcp`.

Reply:
`udp` is defined in Installation guide.
`udp or tcp` accepted